.github/workflows/build.yml

```diff
@@ -171,6 +171,8 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18.x
+      - name: Install prettier
+        run: npm install prettier@2 && npx prettier --version
```

Comment:
I think you can directly do `npx prettier@2` in the Check with prettier

Reply:
npx is a bit redundant here as it combines `npm i` and `npm run`

Reply:
Had to put `npx prettier@2` in all place of use, not sure if it's better or worse for future additions.